integration_tests/sparsearray/src/test/java/org/robolectric/integrationtests/sparsearray/SparseArraySetTest.kt

```diff
@@ -0,0 +1,35 @@
+package org.robolectric.integrationtests.sparsearray
+
+import android.util.SparseArray
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+// Despite being a bug for pre-31, we don't want to break 31+ with our change, so we test all SDKs
+@Config(sdk = [Config.ALL_SDKS])
```

Comment:
Does it need?

Reply:
While developing this, I hit a bug where I was overwriting API 31 and breaking it, so I added all API levels to make sure it doesn't do that.